README.md

```diff
@@ -45,12 +45,12 @@ Our [Roadmap](https://doc.stride3d.net/latest/en/contributors/roadmap.html) comm
 1. **Latest [Git](https://git-scm.com/downloads)** with **Large File Support** selected during setup. For convenience, you might also use a Git UI client like [GitExtensions](https://gitextensions.github.io/).
 2. **[.NET 8.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)**
    - Run `dotnet --info` in a console or PowerShell window to see which versions you have installed.
-3. **[Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)** (the Community edition is free) with the following workloads:
-   - **.NET desktop development** with **.NET Framework 4.7.2 targeting pack** (should be enabled by default)
+3. **[Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)** (the Community edition is free) with the following workloads, which can be installed also through [Visual Studio Build Tools](https://aka.ms/vs/17/release/vs_BuildTools.exe) if you would like to use other IDEs:
```

Comment:
The builds tools are mentioned in `Build Stride without Visual Studio` lower in this file, maybe just mention that and setup a link to that header, what do you think ?

Reply:
Yeah, I was trying to figure out how to be crystal clear(er) for non Visual Studio devs, so they can see right away on the top that they would need the same workload.

I think, with your suggestion, this might be a better wordings?

"other IDEs" would take you to the "Build Stride without Visual Studio" section.

**This below is updated version:**

3. **[Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)** (the Community edition is free), with the following workloads. If you prefer to use [other IDEs](#build-stride-without-visual-studio), you can also install the workloads separately:

Reply:
There might also be an additional requirement when using other IDEs, the path for MSBuild listed under that header, so maybe it's best to just link directly to the header instead, so:

3. [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) (the Community edition is free), with the following workloads. Follow this link if you would rather use [a different IDE or the command line](https://github.com/stride3d/stride/pull/2724#build-stride-without-visual-studio).

What do you think ?

Reply:
Even better, let me finalise it :) Thanks for additional comments :)

Reply:
Done.